test/browser_tests/spec_suites/wagtail-page-templates.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+var LandingPage = require('../page_objects/page_wagtail_templates.js').landing;
```

Comment:
Add spaces inside parens here and throughout to avoid linter errors (try running `gulp lint` to find any other issues)